assets/src/edit-story/components/panels/index.js

```diff
@@ -107,24 +101,20 @@ export function getPanels(elements) {
               return { type, Panel: BackgroundPanel };
             }
             return null;
-          case POSITION:
-            return { type, Panel: PositionPanel };
           case SCALE:
             return { type, Panel: ScalePanel };
-          case ROTATION_ANGLE:
-            return { type, Panel: RotationPanel };
           case SIZE:
             return { type, Panel: SizePanel };
           case FILL:
             return { type, Panel: FillPanel };
           case BACKGROUND_COLOR:
-            return { type, Panel: BackgroundColorPanel };
+            return { type, Panel: StylePanel };
```

Comment:
These two lines are pretty weird now. The constant should better reflect which panel it's asking for.

Reply:
Done.